README.md

```diff
@@ -16,10 +16,20 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 ### To run a kind cluster locally 
 
 ```sh
-make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true
+make cluster-up
+kind get kubeconfig > tmp/kubeconfig
```

Comment:
Same here as well in case we want to export the kubeconfig for microshift

Reply:
sure but the point is that as it currently stands, defining these here have no value, so beyond the scope of this pr :)

Reply:
@vprashar2929 I feel the suggestion is valuable, could you please open a PR for this so that these env vars are  honoured ?

Reply:
Sure sounds good. I will open a PR related to this.